packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1764,6 +1844,8 @@ class EditableText extends StatefulWidget {
     properties.add(DiagnosticsProperty<bool>('enableIMEPersonalizedLearning', enableIMEPersonalizedLearning, defaultValue: true));
     properties.add(DiagnosticsProperty<bool>('enableInteractiveSelection', enableInteractiveSelection, defaultValue: true));
     properties.add(DiagnosticsProperty<SpellCheckConfiguration>('spellCheckConfiguration', spellCheckConfiguration, defaultValue: null));
+    properties.add(DiagnosticsProperty<List<String>>('contentCommitMimeTypes', contentCommitMimeTypes,
+        defaultValue: onContentCommitted == null ? null : kDefaultContentCommitMimeTypes));
```

Comment:
I think this is more inline with Flutter's style.  Or just make it all one line like the ones above.

```dart
    properties.add(DiagnosticsProperty<List<String>>(
      'contentCommitMimeTypes',
      contentCommitMimeTypes,
      defaultValue: onContentCommitted == null ? null : kDefaultContentCommitMimeTypes,
    ));
```